### ISSUE TITLE
Normalizing samples / more gain / trim?

### ISSUE THREAD

**Issue by unfa** [openAVproductions/openAV-Fabla2#63]
I often wish I could have 12 more decibels up on the Gain knob to better match the volume of my samples.

Maybe a way to normalize them would suffice?

Maybe dragging up/down in the waveform window could change gain/trim levels?

![image](https://user-images.githubusercontent.com/491704/29580322-ea0f675e-8775-11e7-8d12-a3f035fcb6e4.png)

![image](https://user-images.githubusercontent.com/491704/29580318-e43eed18-8775-11e7-8100-3361655fd62b.png)

Or maybe the Gain should be changed to a number filed that has no limits and the waveform drag up/down would just be an easier way to adjust that?

What do you think?

**Comment by harryhaaren** [comment]
Yes the normailzation/gain of samples needs to be improved, thanks for noting here.
